addons/website_crm_partner_assign/tests/test_partner_assign.py

```diff
@@ -150,6 +150,12 @@ def test_partner_assign(self):
         except:
             pass
 
+    def test_partner_opportunity_count(self):
+        """ Test that if opportunity is both created and assigned to the same partner.
+            then the opportunity count should be considered as one only.
+        """
```

Comment:
Something like this,
```suggestion
        """ Test the count of opportunities, when the same partner is set for both
        `partner_id` and `partner_assigned_id`.
        """
```